### DISCUSSION TITLE
Add a .vale.ini file

### DISCUSSION THREAD

**Discussion by ccoVeille** [jargonLint/jargonLint#65] [Ideas]
https://github.com/redhat-documentation/vale-at-red-hat/blob/main/.vale.ini

I like how they disable check in:
- URLs
- backtip code block

**Comment by alexvonme** [comment]
I changed the topic since we don't have a .vale.ini file yet. We can add it in the ``vale/`` directory.

To my knowledge, code blocks are ignored by default. We could add URL escape options, but I have a different suggestion.
Why not avoid everything between squared brackets in .md files? That way we disable checking any links regardless of whether they are written like ``https://example.com``, ``www.example.com`` or ``example.com``.
